src/Randomizer.Data/Configuration/Yaml/Sassy/hint_tiles.yml

```diff
@@ -3,6 +3,10 @@
   - Text: 'I guess you''re too lazy to go there yourself? Fine. It says {0}'
   - Text: 'Can''t read it yourself? It says {0}'
   - Text: 'From what I''ve heard, it says {0}'
+  - Text: It says visit Shaktool.<break strength='1s'/>Fine, it says {0}.
```

Comment:
Is this the right syntax? I thought strength was for the relative terms. I think duration is a different attribute? I could be wrong.

Reply:
I'll double check in the morning my time. I think I just copied and pasted from elsewhere, but nothing to say where I pulled it from isn't broken.